tests/templates/message_templates/event_online_receipt_text.tpl

```diff
@@ -11,9 +11,6 @@ isOnWaitlist:::{$isOnWaitlist}
 isRequireApproval:::{$isRequireApproval}
 {/if}
 participant_status:::{$participant_status}
-{if !empty($pricesetFieldsCount)}
```

Comment:
This is in the test template

The variable is still assigned to the template but we aren't checking this & we don't intend to - we can check the output if we want to add testing